storage/tsdb/tsdb.go

```diff
@@ -180,8 +185,16 @@ func Open(path string, l log.Logger, r prometheus.Registerer, opts *Options) (*t
 		}
 	}
 
+	walSegmentSize := int(opts.WALSegmentSize)
+	if walSegmentSize != 0 {
+		if walSegmentSize < 10*1024*1024 || walSegmentSize > 256*1024*1024 {
+			return nil, errors.New("tsdb.Options.WALSegmentSize must be set between 10MB and 256MB")
```

Comment:
maybe best to have this after this line.
https://github.com/prometheus/prometheus/blob/master/cmd/prometheus/main.go#L562

Reply:
yep i agress, done

Reply:
Maybe i could add a test to check if the app stop the right way if the flag value is set out of authorized bounds ?

Reply:
yeah that is not a bad idea.

Reply:
Done !